architecture/core_concepts/routes.adoc

```diff
@@ -322,9 +322,19 @@ In order for services to be exposed externally, an {product-title} route allows
 you to associate a service with an externally-reachable host name. This edge
 host name is then used to route traffic to the service.
 
-When two routes claim the same host, the oldest route wins. If additional routes
-with different path fields are defined in the same namespace, those paths will be
-added. If multiple routes with the same path are used, the oldest takes priority.
+When multiple routes from different namespaces claim the same host,
```

Comment:
Well, there's also the case to consider that 2 routes in the same namespace claiming the same host - so maybe drop the part about `from different namespaces` and `claims it for the namespace`.